language-support/scala/bindings/src/main/scala/com/digitalasset/ledger/client/binding/binding/ContractTypeCompanion.scala

```diff
@@ -36,4 +36,11 @@ abstract class ContractTypeCompanion[T] extends ValueRefCompanion {
       arguments: Option[rpcvalue.Value],
   )(implicit exon: ExerciseOn[ExOn, T]): Primitive.Update[Out] =
     Primitive.exercise(this, receiver, choiceId, arguments getOrElse Value.encode(()))
+
+  protected final def ` exercise`[ExOn, Out](
```

Comment:
We can just delete the argument from the sole ` exercise` method, it is not part of the public API.